src/main/java/gim/ui/ExerciseCard.java

```diff
@@ -51,7 +51,13 @@ public ExerciseCard(Exercise exercise, int displayedIndex) {
         weight.setText("Weight: " + exercise.getWeight().value + "kg");
         reps.setText("Reps: " + exercise.getReps().value);
         sets.setText("Sets: " + exercise.getSets().value);
-        dates.getChildren().add(new Label(exercise.getDateString()));
+
+        String dayString = exercise.getDate().getDayString();
+        // Convert "SATURDAY" to "Saturday"
+        dayString = dayString.charAt(0) + dayString.substring(1).toLowerCase();
```

Comment:
Could this be abstracted perhaps as a method in Date class? Also, for the formatting is it possible to remove the comma and just have a space, i.e. dayString + " " + exercise.getDateString(). This is for UI purposes, thanks.